src/components/Workspace.js

```diff
@@ -498,7 +499,7 @@ class Workspace extends Component {
         }
         <div ref={(editorDiv) => { this.editorDiv = editorDiv; }} id="blocklyDiv">
           <div style={{ position: 'absolute', bottom: 30, right: 100 }}>
-            { React.cloneElement(children, { isConnected: !!rover.rover }) }
+            {React.cloneElement(children, { isConnected: !!rover.rover })}
```

Comment:
Did you editor make this change? I'm surprised the linter was ok with it because it generally wants a space on each end

Reply:
Strange. My eslint autoformatter in VSCode must have done it. Usually it matches what the linter does. This must be an ambigious rule.
I re-added the spaces.